File: jvm/undertest-junit4/src/test/java/undertest/junit4/UT_WithinMethodGCTest.java
package undertest.junit4;

import org.junit.Test;
import static com.diffplug.selfie.Selfie.expectSelfie;

public class UT_WithinMethodGCTest {
    @Test
    public void selfie2() {
//  @Test protected void selfie() {
//    expectSelfie("root").toMatchDisk();
    expectSelfie("oak").toMatchDisk("leaf");
    }

    @Test
    public void secondMethod() {
        expectSelfie("abc123").toMatchDisk();
    }
}

File: undertest-junit5/src/test/kotlin/undertest/junit5/UT_WithinMethodGCTest.kt
package undertest.junit5
// spotless:off
import com.diffplug.selfie.Selfie.expectSelfie
import kotlin.test.Test

// spotless:on

class UT_WithinMethodGCTest {
  @Test fun selfie2() {
//  @Test fun selfie() {
//    expectSelfie("root").toMatchDisk()
    expectSelfie("oak").toMatchDisk("leaf")
  }
//
  @Test fun secondMethod() {
    expectSelfie("abc123").toMatchDisk()
  }
}
